python/ray/_private/state.py

```diff
@@ -189,7 +167,7 @@ def node_table(self):
             }
             node_info["alive"] = node_info["Alive"]
             node_info["Resources"] = (
-                self.node_resource_table(node_info["NodeID"])
+                {key: value for key, value in item.resources_total.items()}
```

Comment:
This line looks weird to me. I believe it's better to copy or deepcopy a dict directly:
```python
node_info["Resources"] = item.resources_total.copy()
# or deepcopy
node_info["Resources"] = copy.deepcopy(item.resources_total)
```